test/integration/footer-links.test.js

```diff
@@ -0,0 +1,142 @@
+const SeleniumHelper = require('./selenium-helpers.js');
+
+const {
+    clickText,
+    buildDriver
+} = new SeleniumHelper();
+
+let remote = process.env.SMOKE_REMOTE || false;
+let rootUrl = process.env.ROOT_URL || 'https://scratch.ly';
+
+if (remote) {
+    jest.setTimeout(60000);
+} else {
+    jest.setTimeout(10000);
+}
+
+let driver;
+
+describe('www-integration footer links', () => {
+    beforeAll(async () => {
+        driver = await buildDriver('www-integration footer links');
+    });
+
+    beforeEach(async () => {
+        await driver.get(rootUrl);
+    });
+
+    afterAll(async () => await driver.quit());
+
+    // ==== About Scratch column ====
+
+    test('click About Scratch link', async () => {
+        await clickText('About Scratch');
+        let url = await driver.getCurrentUrl();
+        expect(url).toMatch(/about\/?$/);
```

Comment:
We could be stricter here, right now we only ensure the URL ends in `about`.
```suggestion
        let url = await driver.getCurrentUrl();
        let pathname = (new URL(url)).pathname;
        expect(pathname).toMatch(/^\/about\/?$/);
```

Reply:
Okay, I updated it for all of the tests.